services/scripts.py

```diff
@@ -1,15 +1,49 @@
 # Music recommendation application
 import csv
-from .cfg import ARTIST_INDEX, LIKES_INDEX
+
+from cfg import TableEnum
+
+
+class Song:
+    def __init__(self, row: list[str]) -> None:
+        self.id = int(row[TableEnum.ID.value])
+        self.name = row[TableEnum.Track.value]
+        self.album = row[TableEnum.Album.value]
+        self.artist = row[TableEnum.Artist.value]
+
+
+class DataBase:
+    def __init__(self, filename):
+        self.db = {}
+
+        with open(filename, encoding='utf-8-sig') as file:
+            _ = file.readline()
+            reader = csv.reader(file)
+
+            for row in reader:
+                self.add_song(Song(row))
+
+    def add_song(self, song: Song) -> None:
+        self.db[song.id] = song
```

Comment:
You need to check that song with this id does not exist yet, if it does, this way of adding son will override it.

Reply:
Sounds strange, because ID is unique parameter.